patches/tModLoader/Terraria/ID/ItemID.cs.patch

```diff
@@ -334,6 +334,12 @@
 +		/// </remarks>
  		public static bool[] IsAPickup = Factory.CreateBoolSet(58, 184, 1734, 1735, 1867, 1868, 3453, 3454, 3455);
 +
++		/// <summary> Indicates that an item is to be filtered under the "Tools" filter in Journey Mode's duplication menu.
++		/// <br/> Useful for manually setting an item to be filtered under the "Tools" filter, if your item does not meet the automatic criteria for the "Tools" filter.
++		/// <br/> See (<see cref="GameContent.Creative.ItemFilters.Tools.FitsFilter(Item)"/>) to check if your item meets the automatic criteria.
++		/// </summary>
++		public static bool[] ToolsFilter = Factory.CreateBoolSet(509, 850, 851, 3612, 3625, 3611, 510, 849, 3620, 1071, 1543, 1072, 1544, 1100, 1545, 50, 3199, 3124, 5358, 5359, 5360, 5361, 5437, 1326, 5335, 3384, 4263, 4819, 4262, 946, 4707, 205, 206, 207, 1128, 3031, 4820, 5302, 5364, 4460, 4608, 4872, 3032, 5303, 5304, 1991, 4821, 3183, 779, 5134, 1299, 4711, 4049, 114);
```

Comment:
Probably would be a good idea to move this to `ItemID.TML.cs`

Reply:
> Probably would be a good idea to move this to `ItemID.TML.cs`

Curious, what is the difference between them?

Reply:
Ah ok, makes sense, I will make the necessary changes.